### ISSUE TITLE
Support HoloEverywhere & ActionbarSherlock

### ISSUE THREAD

**Issue by agarwlGaurav** [SimonVT/android-menudrawer#49]
Does menudrawer support HoloEverywhere & ActionbarSherlock?

Any sample app?

**Comment by SimonVT** [comment]
Should work fine.

**Comment by agarwlGaurav** [comment]
1. I imported HoloEverywhere and ActionbarSherlock in menudrawer Sample "ListActivitySample".
2. Changed android:theme="@style/Holo.Theme in Android Manifest.
3. Changed following imports in "ListActivitySample"

`import org.holoeverywhere.ArrayAdapter;
import org.holoeverywhere.app.ListActivity;
import org.holoeverywhere.widget.ListView;
import org.holoeverywhere.widget.TextView;
import org.holoeverywhere.widget.Toast;`
1. Below method is giving the error the method "cannot overwrite, overwritten method is final "

`public boolean onOptionsItemSelected(MenuItem item) {`
